codalab/worker/local_run/docker_image_manager.py

```diff
@@ -125,7 +125,7 @@ def _cleanup(self):
 
     def get(self, image_spec):
         """
-        Request the docker image, starting its download if it's not available on the system
+        Always request the newest docker image from Dockerhub if it's not in downloading thread and return the current downloading status
```

Comment:
Please write more comprehensive documentation documenting the logic about the possible cases.